### PR TITLE
Lesters/add batch norm tf case

### PR DESCRIPTION
**Pull request by lesters**
Replaced the 3-layer test case with this more complex network.

Note that the PlaceholderWithImport adds a macro expression to the TensorFlowModel, but this has not been wired into config yet. For this case this is not a problem, because the constant this is used for is evaluated during TensorFlow import and then not used in the resulting expression (see the 'select' operation). This might be true in most cases for such TF import, but we can wire this in anyway just in case.

### CODE REVIEW

**Comment by lesters** [comment]
Also, some of the functions that are imported and implemented in OperationMapper could probably be implemented as non-primitive tensor functions as well.

**Review by bratseth — APPROVED** [review]
Great
